File: src/main/java/com/github/tgnthump/cfnnagintellijplugin/services/MyProjectService.java
package com.github.tgnthump.cfnnagintellijplugin.services;

import com.github.tgnthump.cfnnagintellijplugin.MyBundle;
import com.intellij.openapi.project.Project;

public class MyProjectService {

    private final Project project;

    public MyProjectService(Project project) {
        this.project = project;
        System.out.println(MyBundle.message("projectService", project.getName()));
    }
}

File: src/main/kotlin/com/github/tgnthump/cfnnagintellijplugin/services/MyProjectService.kt
package com.github.tgnthump.cfnnagintellijplugin.services

import com.intellij.openapi.project.Project
import com.github.tgnthump.cfnnagintellijplugin.MyBundle

class MyProjectService(project: Project) {

    init {
        println(MyBundle.message("projectService", project.name))
    }
}
